modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/en-US.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html charset=UTF-8" />
+</head>
+
+<body>
+	<div class="header-logo" style="padding: 40px; text-align: center; margin: auto;"><img alt="Liferay Logo" class="logo"
+			src="http://localhost:18080/documents/d/PRM/liferay_logo_image-png" width="200" />
+	<div class="section-line" style="margin: auto; margin-top: 20px; margin-bottom: 20px; background-color: #6b6c7e; height: 1px; width: 592px;">
+			 </div>
+	</div>
+
+	<div class="main-container" style="margin: 0% 33%;">
+		<div>
+			<h1 class="text-primary" style="color: #377cff; font-size: 24px; margin-bottom: 16px;">Liferay MDF Request
+				Completed</h1>
```

Comment:
do we normally do new lines like this? like `Liferay MDF Request` on one line but `Completed` on the next? it seems weird to cut it mid phrase. 

i think it should be one of these options:
```
<h1>
     text
</h1>
```
```
<h1>text</h1>
```

Reply:
Fixed!